### ISSUE TITLE
The Youtube's progress bar is wrongly actionable outside its area

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [MozillaReality/FirefoxReality#2463]
**[Affected Versions]:**
- Firefox Reality version 8-33452219 (bfee54ee)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- HTC Vive Focus Plus - Android 7.1.2

**[Prerequisites]:**
- The Firefox Reality browser is opened.

**[Steps to reproduce]:**
1. Navigate to a Youtube video.
2. Click outside the video's progress bar boundaries.
3. Observe the behavior.

**[Expected results]:**
- The progress bar is not actionable.

**[Actual results]:**
- The progress bar is actionable.

**[Notes]:**
- The actionable area of the progress bar is also affecting the videos controls buttons displayed under the progress bar.
- Needed by #1760.
- Attached is a screen recording of the issue: [link](https://drive.google.com/open?id=13sYSZC0Aw3RYDdsyVmbqiq_LQztXHy3D).

**Comment by jvonitter** [comment]
@MortimerGoro any idea if this is our problem or theirs?

**Comment by MortimerGoro** [comment]
@MortimerGoro I'm working on fixing the layout problems in the WebExtension refactor, some of the problems are related to using a viewport that youtube doesn't like. I think this issue will be fixed too, I'll check it.

**Comment by Softvision-RemusDranca** [comment]
I have verified this issue and it's still reproducible with the latest Firefox Reality build 8-33521753 (379b8f1f) from master. Also, the range of the clickable area seems to be interfering with the area of the controls buttons. For example when I clicked the full screen button the progress of the video was moved to that position and also the video entered in theater mode.

Tested on Oculus Quest (Android 7.1.1), Oculus Go (Android 7.1.2), HTC Vive Focus Plus (Android 7.1.2) and HTC Vive Focus (Android 7.1.2).

Considering this I will reopen this issue and also change the severity of it to "Normal".
